setup.py

```diff
@@ -4,18 +4,25 @@
 
 from chirp import CHIRP_VERSION
 
-desktop_files = glob("share/*.desktop")
-image_files = glob('share/*.png')
-stock_configs = glob("stock_configs/*")
-
 setup(name='chirp',
-      descrption='A cross-platform cross-radio programming tool',
-      packages=find_packages(exclude=["tests", "tests.*"]),
+      description='A cross-platform cross-radio programming tool',
+      packages=find_packages(include=["chirp*"]),
+      include_package_data=True,
       version=CHIRP_VERSION,
       url='https://chirp.danplanet.com',
-      python_requires=">=3.3,<4",
-      install_required=['wxPython', 'serial', 'six', 'future'],
+      python_requires=">=3.7,<4",
+      install_requires=[
+          'pyserial',
+          'six',
+          'future',
+          'importlib-resources;python_version<"3.10"'
+      ],
+      extras_require={
+          'wx': ['wxPython'],
+          'gtk': ['PyGObject']
+      },
       entry_points={
           'console_scripts': ["chirp=chirp.wxui:chirpmain"],
       },
+      scripts=['chirpw'],
```

Comment:
This should be removed, and the chirpw from manifest. It "works" in that it loads from time to time but things break, I don't want to support it, and plan to remove it from the tree, again, once we're no longer keeping the py2 and py3 branches in sync. Last I checked it didn't even start due to changes sync'd from master. It confuses people having both and I want to avoid that. That also means the gtk extras target can go.

Reply:
I've been using that since the last commit in the py3 branch in mercurial with only Python syntax changes, using PyGObject3. Even under that (and GTK 3), it is significantly lighter than and has working multi-select and copy-paste functionality compared to the wxPython interface.

Reply:
It's massively slower in my experience, I'm shocked that you find it faster. There's been over a year of development on the wx ui since the last commit on the mercurial branch. There are still some gaps, but I'm working to close them. If you find any not on the list I sent to the mailing list, feel free to bring them up.

On Windows and macos, wx provides a far superior experience for people (and native widgets), which again, represent 99% of the users :)